test/built-ins/ArrayBuffer/prototype/byteLength/return-bytelength.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 24.1.4.1
+esid: sec-get-arraybuffer.prototype.bytelength
+description: Return value from [[ByteLength]] internal slot
+info: >
+  24.1.4.1 get ArrayBuffer.prototype.byteLength
+
+  ...
+  5. Let length be the value of O's [[ArrayBufferByteLength]] internal slot.
+  6. Return length.
+---*/
+
+var ab1 = new ArrayBuffer(0);
+assert.sameValue(ab1.byteLength, 0);
```

Comment:
Ideally, this sample should be constructed with no arg as `var ab1 = new ArrayBuffer();`, but that needs tc39/ecma262#410 to be merged first.